src/ItemList.h

```diff
@@ -72,21 +73,70 @@ class ItemList : public MenuItem {
      *
      * @return The the callback bound to the current item.
      */
-    fptrInt getCallbackInt() override { return callback; }
+    fptrInt getCallbackInt() { return callback; }
 
     /**
      * @brief Returns the total number of items in the list.
      *
      * @return The total number of items in the list.
      */
-    uint8_t getItemCount() override { return itemCount; };
+    uint8_t getItemCount() { return itemCount; };
 
     /**
      * @brief Returns a pointer to the array of items.
      *
      * @return A pointer to the array of items.
      */
-    String* getItems() override { return items; }
+    String* getItems() { return items; }
+
+    void enter() override {
+        if (display->getEditModeEnabled()) {
+            return;
+        }
+        display->setEditModeEnabled(true);
+    };
+
+    void back() override {
+        if (!display->getEditModeEnabled()) {
+            return;
+        }
+        display->setEditModeEnabled(false);
+        if (callback != NULL) {
+            callback(itemIndex);
+        }
+    };
+
+    void left() override {
+        if (!display->getEditModeEnabled()) {
+            return;
+        }
+        uint8_t previousIndex = itemIndex;
+        itemIndex = constrain(itemIndex - 1, 0, (uint16_t)(itemCount)-1);
+        if (previousIndex != itemIndex) {
+            printCmd(F("LEFT"), items[itemIndex].c_str());
+            MenuItem::draw();
+        }
+    };
+
+    void right() override {
+        if (!display->getEditModeEnabled()) {
+            return;
+        }
+        uint8_t previousIndex = itemIndex;
+        itemIndex = constrain((itemIndex + 1) % itemCount, 0, (uint16_t)(itemCount)-1);
+        if (previousIndex != itemIndex) {
+            printCmd(F("RIGHT"), items[itemIndex].c_str());
+            MenuItem::draw();
+        }
+    };
```

Comment:
Override instead `up` and `down` so that the behavior is consistent with ItemInput.
So, only `ItemInput` would use left and right, I discovered that the input rotary example did not work because it calls `left` and `right` when in edit mode.

Would really be good if they all do similar actions with `up` and `down`, this will also slim down the RotaryNavConfig cuz they'll no longer be a need to call `left` when in edit and `down` when not. It will always be `down`.

Doing this also makes `ItemInputChasetSimple` useless.

I will provide a cloud patch in my last comment so you can see the changes which I made for all the examples to work properly.

```suggestion
    void down() override {
        if (!display->getEditModeEnabled()) {
            return;
        }
        uint8_t previousIndex = itemIndex;
        itemIndex = constrain(itemIndex - 1, 0, (uint16_t)(itemCount)-1);
        if (previousIndex != itemIndex) {
            printCmd(F("LEFT"), items[itemIndex].c_str());
            MenuItem::draw();
        }
    };

    void up() override {
        if (!display->getEditModeEnabled()) {
            return;
        }
        uint8_t previousIndex = itemIndex;
        itemIndex = constrain((itemIndex + 1) % itemCount, 0, (uint16_t)(itemCount)-1);
        if (previousIndex != itemIndex) {
            printCmd(F("RIGHT"), items[itemIndex].c_str());
            MenuItem::draw();
        }
    };
```